class.jetpack.php

```diff
@@ -6354,4 +6354,13 @@ function jetpack_user_col_style() {
 			</style>
 		<?php }
 	}
+
+	/**
+	 * Return blog locale as a two-character string
+	 *
+	 * @return string The blog's locale, e.g. 'en' or 'it'
+	 */
+	public static function get_blog_locale() {
```

Comment:
I think you are probably correct. In my case this is used to get to different subdomain URLs, e.g. `en.wordpress.com`. I don't know where the master list of these URLs is kept, but seeing as further searching reveals there are many >2 character shortcodes for locales, I'm going to update the code with your suggestion.